soco/core.py

```diff
@@ -949,17 +956,88 @@ def dialog_mode(self, dialog_mode):
         :raises NotSupportedException: If the device does not support
         dialog mode.
         """
+        self.renderingControl.set_eq_variable("DialogLevel", int(dialog_mode))
+
+    @property
+    def surround_mode(self) -> Optional[bool]:
```

Comment:
General comment: while it's great to see type hints, this is obviously not something found elsewhere in the SoCo code. However, I'm going to propose leaving them in, trading off their benefit for the inconsistency.

Note that there aren't type checks in the check jobs at present.

Reply:
Yeah, I realized they are missing, but the good thing with them is that there is no need to do a full conversion for them to be useful.